src/main/java/com/autosale/shop/service/impl/UserSessionServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package com.autosale.shop.service.impl;
+
+import com.autosale.shop.model.User;
+import com.autosale.shop.model.UserSession;
+import lombok.RequiredArgsConstructor;
+import org.springframework.cache.annotation.CacheEvict;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.stereotype.Service;
+
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class UserSessionServiceImpl implements com.autosale.shop.service.UserSessionService {
```

Comment:
use import  com.autosale.shop.service

Reply:
just a side effect after automatically extracting the interface from the class